config/reports/fietsSubsidieRequestsReport.js

```diff
@@ -0,0 +1,71 @@
+import { generateReportFromData } from '../helpers.js';
+import { querySudo as query } from '@lblod/mu-auth-sudo';
+
+export default {
+  cronPattern: '0 40 23 * * *',
+  name: 'fietsSubsidieRequestsReport',
+  execute: async () => {
+    const reportData = {
+      title: 'List of requests of fiets subsidies',
+      description: 'All requests for bike subsidies that have been sent with their related information',
+      filePrefix: 'fietsSubsidieRequestsReport'
+    };
+    console.log('Generate Fiets Subsidie Requests Report');
+    const queryString = `
+      PREFIX pav: <http://purl.org/pav/>
+      PREFIX prov: <http://www.w3.org/ns/prov#>
+      PREFIX subsidie: <http://data.vlaanderen.be/ns/subsidie#>
+      PREFIX transactie: <http://data.vlaanderen.be/ns/transactie#>
+      PREFIX m8g: <http://data.europa.eu/m8g/>
+      PREFIX dct: <http://purl.org/dc/terms/>
+      PREFIX adms: <http://www.w3.org/ns/adms#>
+
+      SELECT DISTINCT ?submissionDate ?bestuurseenheid ?subsidiemaatregelConsumptie ?status ?iban
+      WHERE {
+        ?subsidiemaatregelConsumptie
+          transactie:isInstantieVan <http://lblod.data.gift/concepts/70cc4947-33a3-4d26-82e0-2e1eacd2fea2> ;
+          adms:status/skos:prefLabel ?status ;
+          dct:modified ?submissionDate ;
+          m8g:hasParticipation ?participation ;
+          dct:source ?applicationForm .
+        ?bestuur m8g:playsRole ?participation ;
+          skos:prefLabel ?bestuurseenheid .
+
+        OPTIONAL { ?applicationForm <http://schema.org/bankAccount>/<http://schema.org/identifier> ?iban . }
+
+        FILTER EXISTS {
+          ?applicationForm dct:isPartOf ?step .
+          ?step dct:references <http://data.lblod.info/id/subsidieprocedurestappen/52f0b7dd244e42e0cda83804508e2e89d94ed098f3df8b4f9913a14f2be2423d> .
+        }
```

Comment:
would it make sense to add:
```
        FILTER NOT EXISTS {
          ?applicationForm adms:status <http://lblod.data.gift/concepts/6b7ae118-4653-48f2-9d9a-4712f8c30da9>. # new
        }```

Reply:
It would I think, maybe not on step 1 because they might want to see if the bestuur has at least started something, but on step 2 indeed it'll just be blank

Reply:
It is for step 2 right? Because, now as soon as they click on the next step, even if the step 1 is not ok, they are taken in the report.

Reply:
Yes. What I meant is that now, a step 1 will also be "New" until the first time it's saved, but we might want to keep it in the report for step 1.
Step 2 --> Only "Concept" ones are interesting